app/validation/validator.py

```diff
@@ -153,8 +153,14 @@ def _validate_blocks(  # noqa: C901 pylint: disable=too-many-branches
                               'ListCollectors')
             elif block['type'] == 'RelationshipCollector':
                 errors.extend(self._validate_relationship_list_exists(block))
-                errors.extend(self._validate_relationship_collector_contains_single_relationship_answer(block))
-                errors.extend(self._validate_relationship_collector_answer_type(block))
+
+                if 'question_variants' in block:
+                    # Variants are checked to ensure they are consistent
+                    answers = block['question_variants'][0]['question']['answers']
```

Comment:
It's probably worth checking all of the variants here. Unlikely to cause any major issues, but would save some time if you had a problem in your second variant.

Reply:
I thought we had a method for getting all answers on a question, but I can't see it now.